README.md

```diff
@@ -705,3 +705,93 @@ agentChatSession.cleanUpOnParticipantDisconnect();
 Cleans up all event handlers.
 
 Applies only for `AgentChatSession`. See `connect.ChatSession.create()` for more info.
+
+## Handle Browser Refresh
+
+> For latest documentation, please follow instructions in ["Admin guide: Enable real-time chat message streaming"](https://docs.aws.amazon.com/connect/latest/adminguide/chat-message-streaming.html)
+
+Reconnect to an active chat after refreshing the browser. Call the `CreateParticipantConnection` on refresh with the same `ParticipantToken` generated from `ShatChatContact`.
+
+### Reference
+
+- StartChatContact: [Documentation](https://docs.aws.amazon.com/connect/latest/APIReference/API_StartChatContact.html)
+- CreateParticipantConnection: [Documentation](https://docs.aws.amazon.com/connect-participant/latest/APIReference/API_CreateParticipantConnection.html)
+- "Enable real-time chat message streaming": [Documentation](https://docs.aws.amazon.com/connect/latest/adminguide/chat-message-streaming.html)
+
+### Walkthrough
+
+1) Initialize the chat session with a [`StartChatContact`](https://docs.aws.amazon.com/connect/latest/APIReference/API_StartChatContact.html) request:
+
+
+2) Save the `ParticipantToken` from the `StartChatContact` request:
+
+```http
+HTTP/1.1 200
+Content-type: application/json
+
+{
+  "ContactId": "string",
+  "ParticipantId": "string",
+  "ParticipantToken": "string"
+}
+```
+
+3) Refresh the browser
+
```

Comment:
Lets explain Step 4 on how this will be done using Chat.js

When Customer calls `ChatSession.create` similar to how we do it here https://code.amazon.com/packages/AmazonConnectChatInterface/blobs/38fa887ee4240eaa08ef1a7a86e31ee35cd38166/--/src/components/Chat/ChatSession.js#L33 in ChatInterface

Customer can pass `chatDetails` args to ChatJS here https://code.amazon.com/packages/AmazonConnectChatJS/blobs/fc12ed4fb976ec3a753dff268eb0edc5b19d04b5/--/src/core/chatSession.js#L224,L215
eg ChatDetails object 
```
{
getConnectionToken: () => connectionToken,
participantToken: xxxx,
}
```
used here https://code.amazon.com/packages/AmazonConnectChatJS/blobs/a1356d6b557a7256904a811e5eb18400c61b3cc5/--/src/core/chatController.js#L38,L44

participantToken is then passed for calling `createParticipantConnection` here https://code.amazon.com/packages/AmazonConnectChatJS/blobs/e4fe1242701bd46ed100441951f565d88f21c662/--/src/core/connectionHelpers/connectionDetailsProvider.js#L66

Reply:
Thank you for reviewing @mhiaror. I have published new revision with improved details.

Here is summarized steps listed:
1) New chat contact is initialized
2) Save the `ParticipantToken` from `StartChatContact`
3) New chat session is established
4) Browser is refreshed. Chat session is disconnected
5) New chat contact is initialized
6) New chat session is established, with `participantToken`
7) ChatJS calls `createParticipantConnection` with `participantToken` (internal)